i18n/enu/out/providers/hoverProvider.i18n.json

```diff
@@ -0,0 +1,3 @@
+{
+	"autolispext.hoverProvider.ambiguous": "Multiple definitions containing the @Global flag"
```

Comment:
@JD-Howard 
When testing on this message I see this:

![image](https://user-images.githubusercontent.com/57521938/192968984-cc503d69-8484-4788-bee4-6989f2f457aa.png)

I guess the string` $(error) `at the beginning is not the expected result?

Reading the source code below I feel that an error icon is expected?
https://github.com/Autodesk-AutoCAD/AutoLispExt/pull/202/files#diff-e176c092ee19a3712e1d5938d6b89322fc76dee085e6cf2ccb55283e9fd6f2b7R154